### PR TITLE
Fixing the corfu_add_node cmdlet

### PR DESCRIPTION
**Pull request by rogermichoud**
## Overview
Reflecting new api of addNode that adds timeout
and poll frequency of state of the workflow.

Why should this be merged: 
Blocks fault-injection testing

## Checklist (Definition of Done):

- [X] There are no TODOs left in the code
- [X] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [X] Change is covered by automated tests
- [X] Public API has Javadoc

### CODE REVIEW

File: corfu_scripts/corfu_add_node.clj [hunk 1]
```diff
@@ -23,7 +25,8 @@ Options:
 (def localcmd (.. (new Docopt usage) (parse *args)))
 
 (defn add-node [endpoint]
-      (println (.. (get-management-view) (addNode endpoint))))
+      (println (.. (get-management-view)
+                   (addNode endpoint 3 (. Duration (ofMinutes 10)) (. Duration (ofSeconds 1))))))
```
Comment (no2chem) [inline corfu_scripts/corfu_add_node.clj]: Shouldn't these parameters come from the docopt string?

**Comment by Maithem** [comment]
can you add remove node as well

**Comment by rogermichoud** [comment]
@no2chem Addressed the comments (Damn it was painful) 

@Maithem Test cycle for cmdlets is too painful and I need more background on how to use Clojure to become somewhat efficient. In the meantime, I will defer your ask to a new pull request. We need this add_node fix for deblocking our testing. Remove node is a bit less urgent.

**Comment by Maithem** [comment]
@rogermichoud ok that's fine, I thought you wanted both for the tests.

**Pull request merged**